cmd/smoketests/main.go

```diff
@@ -101,20 +112,24 @@ func run(ctx context.Context) error {
 		return errors.New("No file ending with '.gfxtrace' found in trace directory")
 	}
 
-	if nbErr > 0 {
-		errStr := "error"
-		if nbErr > 1 {
-			errStr = "errors"
-		}
-		log.I(ctx, "%d %s found, see logs in %s", nbErr, errStr, tmpdir)
+	// Print number of errors
+	errStr := "error"
```

Comment:
You can do the same using "error(s)". In the case nbError == 0 you'll have "found 0 error" which is awkward.

Reply:
Good spot! I kept the logic but fixed the message in case of 0 errors.